devicetypes/qubino/qubino-flush-dimmer-0-10V.src/qubino-flush-dimmer-0-10V.groovy

```diff
@@ -0,0 +1,671 @@
+	syncConfiguration()
+	// Preferences template end
+}
+
+private readConfigurationFromTheDevice() {
+	def commands = []
+	parameterMap.each {
+		state.currentPreferencesState."$it.key".status = "reverseSyncPending"
+		commands += encap(zwave.configurationV2.configurationGet(parameterNumber: it.parameterNumber))
+	}
+	sendHubCommand(commands)
+}
+
+private syncConfiguration() {
+	def commands = []
+	parameterMap.each {
+		if (state.currentPreferencesState."$it.key".status == "syncPending") {
+			commands += encap(zwave.configurationV2.configurationSet(scaledConfigurationValue: getCommandValue(it), parameterNumber: it.parameterNumber, size: it.size))
+			commands += encap(zwave.configurationV2.configurationGet(parameterNumber: it.parameterNumber))
+		} else if (state.currentPreferencesState."$it.key".status == "disablePending") {
+			commands += encap(zwave.configurationV2.configurationSet(scaledConfigurationValue: it.disableValue, parameterNumber: it.parameterNumber, size: it.size))
+			commands += encap(zwave.configurationV2.configurationGet(parameterNumber: it.parameterNumber))
+		}
+	}
+	sendHubCommand(commands)
+}
+
+def getCommandClassVersions() {
+	[
+		0x20:1, // Basic
+		0x25:1, // Switch Binary
+		0x26:3, // Switch Multilevel
+		0x27:1, // Switch All
+		0x59:1, // Association Grp Info
+		0x5A:1, // Device Reset Locally
+		0x70:2, // Configuration
+		0x72:2, // Manufacturer Specific
+		0x73:1, // Powerlevel
+		0x85:2, // Association
+		0x86:1, // Version
+		0x8E:2  // Multi Channel Association
+	]
+}
+
+def configure() {
+	def commands = []
+
+	/*
+		Association Groups:
+		Group 1: Lifeline group (reserved for communication with the primary gateway (hub))
+		Group 2: Basic on/off (status change report for I1 input)
+		Group 3: Start level change/stop (status change report for I1 input). Working only when the Parameter no. 1 is set to mono stable switch type.
+		Group 4: Multilevel set (status change report of the Flush Dimmer 0-10V). Working only when the Parameter no. 1 is set to mono stable switch type.
+		Group 5: Multilevel sensor report (status change report of the analogue sensor)
+		Group 6: Multilevel sensor report (status change report of the temperature sensor)
+	*/
+	commands << zwave.manufacturerSpecificV2.manufacturerSpecificGet().format()
+	commands << zwave.associationV1.associationSet(groupingIdentifier:1, nodeId:[zwaveHubNodeId]).format()
+	commands << zwave.associationV1.associationSet(groupingIdentifier:2, nodeId:[zwaveHubNodeId]).format()
+	commands << zwave.associationV1.associationSet(groupingIdentifier:3, nodeId:[zwaveHubNodeId]).format()
+	commands << zwave.associationV1.associationSet(groupingIdentifier:4, nodeId:[zwaveHubNodeId]).format()
+	commands << zwave.associationV1.associationSet(groupingIdentifier:5, nodeId:[zwaveHubNodeId]).format()
+	commands << zwave.associationV1.associationSet(groupingIdentifier:6, nodeId:[zwaveHubNodeId]).format()
+	commands << zwave.multiChannelV3.multiChannelEndPointGet().format()
```

Comment:
it sounds like, from your description, that some of these should only be set once it's detected that the child device is connected